src/main/java/core/basesyntax/strategy/impl/OperationHandlerBalanceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.OperationHandler;
+
+public class OperationHandlerBalanceImpl implements OperationHandler {
```

Comment:
In each Handler add class field Storage, inject it using constructor when create it in Strategy.